.github/workflows/amplify_integration_tests.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
       - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # 3.1.0
         with:
           persist-credentials: false
+          submodules: true
```

Comment:
question: Is this needed only for android or for other instances of checkout here for other platforms?

Reply:
It's needed for `aft`

Reply:
Makes sense, but this instance of checkout is only for android and all the other platforms in this file use checkout so was thinking they needed same change (as they also use aft in independent runner/job in matrix).